spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/mapping/CassandraPersistentEntity.java

```diff
@@ -70,6 +78,7 @@ default void setTableName(org.springframework.data.cassandra.core.cql.CqlIdentif
 	 */
 	void setTableName(CqlIdentifier tableName);
 
+
```

Comment:
As we have a `setTableName(…)` option, we could introduce `setKeyspaceName(…)` for consistency.